lib/matplotlib/widgets.py

```diff
@@ -595,7 +596,7 @@ def set_active(self, index, state=None):
             raise ValueError("Invalid CheckButton index: %d" % index)
 
         l1, l2 = self.lines[index]
-        target_vis = bool(state) if state is not None else not l1.get_visible()
+        target_vis = bool(state) if state is not _TOGGLE else not l1.get_visible()
```

Comment:
docstring needs to be updated (it still mentions None)

Reply:
done, thanks.